packages/amplify_datastore/lib/amplify_datastore_stream_controller.dart

```diff
@@ -43,58 +43,73 @@ StreamController<DataStoreHubEvent> _controller =
 _onListen() {
   if (eventStream == null) {
     eventStream = channel.receiveBroadcastStream(1).listen((event) {
-      switch (event["eventName"]) {
-        case "ready":
+      switch (event['eventName']) {
```

Comment:
optional nit: I know it's already like this and a little tangential to PR, but can 'eventName' be made a constant? Ideally, lots of serialization keys would be in a constant somewhere,b ut this one is used several times right here, so could just be `const _eventName = 'eventName'` at top of file.